src/status_im/integration_test.cljs

```diff
@@ -155,26 +133,24 @@
            [:wallet.accounts/account-stored]
            (assert-new-account-created) ; assert account was created
            (logout!)
-           (rf-test/wait-for [::logout/logout-method]
-             (assert-logout))))))))
+           (rf-test/wait-for [::logout/logout-method])))))))
 
 (deftest back-up-seed-phrase-test
   (log/info "========= back-up-seed-phrase-test ==================")
   (rf-test/run-test-async
    (initialize-app!)
    (rf-test/wait-for
-     [:setup/initialize-view]
+     [:get-profiles-overview-success]
      (generate-and-derive-addresses!)
      (rf-test/wait-for
        [:multiaccount-generate-and-derive-addresses-success]
-       (assert-multiaccount-loaded)
        (create-multiaccount!)
        (rf-test/wait-for
          [::transport/messenger-started]
          (assert-messenger-started)
          (rf/dispatch-sync [:set-in [:my-profile/seed :step] :12-words]) ; display seed phrase to user
          (rf/dispatch-sync [:my-profile/enter-two-random-words]) ; begin prompting user for seed words
-         (let [ma    @(rf/subscribe [:multiaccount])
+         (let [ma    @(rf/subscribe [:profile/profile])
```

Comment:
I think `ma` is referring to "multiaccount" - can you update  to `profile` instead?

Reply:
yes